edit_here_to_add_blocklists.json

```diff
@@ -916,8 +916,8 @@
     {
       "name": "bongochong (CombinedPrivacy)",
       "category": "Ads, Privacy",
-      "url": ["https://raw.githubusercontent.com/bongochong/CombinedPrivacyBlockLists/master/newhosts-final.hosts"],
-      "filterType": ["b-norm"],
+      "url": ["https://raw.githubusercontent.com/bongochong/CombinedPrivacyBlockLists/master/NoFormatting/cpbl-wildcard-blacklist.txt"],
```

Comment:
Whats the difference between newhosts-final and cpbl? Are they same ?

Reply:
They are the same yes. Rethinkdns uses the exact same wildcard source for their bongochong list and the only difference between the two is the format and the location in the GitHub repo